_sources/functions_in_cpp/mixed_up_code.rst

```diff
@@ -0,0 +1,329 @@
+   =====
+         return 'A';
+   =====
+         return A;  #paired
+   =====
+      }
+   =====
+      else if (percentage >= 80) {
+   =====
+         return 'B';
+   =====
+         return 'B'  #paired
+   =====
+      }
+   =====
+      else if (percentage >= 70) {
+   =====
+      else if (percentage > 70) {  #paired
+   =====
+         return 'C';
+   =====
+      }
+   =====
+      else {
+   =====
+         return 'F';
+   =====
+      }
+   =====
+   }
+
+.. parsonsprob:: functions_in_cpp_mixed_up_code_4
+   :numbered: left
+   :adaptive:
+   :practice: T
+
+   Let's write the code for the ``triangleArea`` function. ``triangleArea``
+   takes two parameters, ``base`` and ``height``. It returns the 
+   area of the triangle using the formula **1/2 * base * height**.
+   Put the necessary blocks of code in the correct order.
+   -----
+   double triangleArea (double base, double height) {
+   =====
+   int triangleArea (double base, double height) {  #distractor
+   =====
+   void triangleArea (double base, double height) {  #distractor
+   =====
+   double triangleArea (base, height) {  #distractor
+   =====
+      double area;  #distractor
+   =====
+      return 0.5 * base * height;
+   =====
+      cout << 0.5 * base * height << endl;  #distractor
+   =====
+   }
+
+.. parsonsprob:: functions_in_cpp_mixed_up_code_5
+   :numbered: left
+   :adaptive:
+   :practice: T
+
+   Let's write the code for the ``cylinderVolume`` function. ``cylinderVolume``
+   takes two parameters, ``radius`` and ``height``. It returns the 
+   volume of the cylinder using the formula **pi * radius * radius * height**.
+   Put the necessary blocks of code in the correct order.
+   -----
+   double cylinderVolume (double radius, double height) {
+   =====
+   void cylinderVolume (double radius, double height) {  #distractor
+   =====
+   double cylinderVolume (radius, height) {  #distractor
+   =====
+      double pi = 3.14;
+   =====
+      return pi * radius * radius * height;
+   =====
+   }
+
+.. parsonsprob:: functions_in_cpp_mixed_up_code_6
+   :numbered: left
+   :adaptive:
+   :practice: T
+
+   Let's write a function called ``scale`` which takes an input ``x`` and scales it by a factor 
+   of 2 is ``x`` is even and a factor of 3 if ``x`` is odd. The function should modify the original
+   ``x`` variable, not return the scaled value.
+   -----
+   void scale(int &x) {
+   =====
+   void scale(int x) {  #distractor
+   =====
+   int scale(int x) {  #distractor
+   =====
+      if (x % 2 == 0) {
+   =====
+      if (x % 2) {  #paired
+   =====
+         x = x * 2;
+   =====
+      }
+   =====
+      return x * 2; #distractor
+   =====
+      else {
+   =====
+         x = x * 3;
+   =====
+         return x * 3;  #distractor
+   =====
+      }
+   =====
+   }
+
+.. parsonsprob:: functions_in_cpp_mixed_up_code_7
+   :numbered: left
+   :adaptive:
+   :practice: T
+
+   Let's write the code for the ``isDoubleDigit`` function. ``isDoubleDigit``
```

Comment:
It looks like this problem hasn't been deleted all of the way - the problem description is still there, but the Parson's problem doesn't have any blocks in it.

Reply:
Was this meant to be deleted? For this question the option to use an else block was removed, not the entire question. I can delete the entire question if that's needed.

Reply:
Oh that's fine then! There's still a problem with how it shows up - I can see the question, but I can't see any blocks in the Parson's problem.

Reply:
That is a problem! When I build these pages everything shows up, but I have the files locally. Do you get any errors or warnings about these problems when you go to build the pages? Also, are the following questions you noted were not deleted correctly having the same issue?

Reply:
Hmm... I don't get any errors or warnings. I'm using `runestone build` to build it locally (after pulling all of the changes from this pull request). Yeah, all of the questions I've noted are having the same problem. They show up like this:

![image](https://user-images.githubusercontent.com/4933398/128924897-cb7bc238-e6de-477e-bd4d-323006f10b87.png)

Reply:
When I look at `_sources/functions_in_cpp/mixed_up_code.rst`, I still see the code from the Parson's problem for `isDoubleDigit` - can we delete it from there?

Reply:
I can go ahead and delete those Parson's problems from the functions_in_cpp chapter while I figure out why these in particular aren't loading correctly.